rustls/tests/common/mod.rs

```diff
@@ -337,9 +348,9 @@ impl KeyType {
         )
     }
 
-    fn get_crl(&self, role: &str) -> CertificateRevocationListDer<'static> {
+    fn get_crl(&self, role: &str, r#type: &str) -> CertificateRevocationListDer<'static> {
```

Comment:
TIL this syntax works for the case where you want a binding name that overlaps a keyword. Cool :+1: